crates/nu-cli/src/commands/str_/trim/trim_left.rs

```diff
@@ -80,15 +80,22 @@ mod tests {
         let word = string(" andres ");
         let expected = string("andres ");
 
-        let actual = action(&word, Tag::unknown(), None, &trim_left).unwrap();
+        let actual = action(&word, Tag::unknown(), None, &trim_left, ActionMode::Local).unwrap();
         assert_eq!(actual, expected);
     }
     #[test]
     fn trims_custom_chars_from_left() {
         let word = string("!!! andres !!!");
         let expected = string(" andres !!!");
 
-        let actual = action(&word, Tag::unknown(), Some('!'), &trim_left).unwrap();
+        let actual = action(
+            &word,
+            Tag::unknown(),
+            Some('!'),
+            &trim_left,
+            ActionMode::Local,
+        )
+        .unwrap();
```

Comment:
Can you add some test cases for the Global mode, too?

Reply:
It is done :-)